fastestimator/op/numpyop/univariate/rua.py

```diff
@@ -255,6 +259,46 @@ def _apply_solarize(data: np.ndarray, threshold: int) -> np.ndarray:
         return data
 
 
+@traceable()
+class OneOfMultiVar(OneOf):
+    """Perform one of several possible NumpyOps.
+
+    Note that OneOfMultiVar accepts both univariate and multivariate ops and allows the list of passed NumpyOps to have
+    different input and output keys. OneOfMultiVar should not be used to wrap an op whose output key(s) do not already
+    exist in the data dictionary. This would result in a problem when future ops / traces attempt to reference the
+    output key, but OneOfMultiVar declined to generate it. If you want to create a default value for a new key, simply
+    use a LambdaOp before invoking the OneOfMultiVar.
+
+    Args:
+        *numpy_ops: A list of ops to choose between with uniform probability.
+    """
+    def __init__(self, *numpy_ops: NumpyOp) -> None:
+        inputs = to_set(numpy_ops[0].inputs)
+        outputs = to_set(numpy_ops[0].outputs)
+        mode = numpy_ops[0].mode
+        self.in_list = numpy_ops[0].in_list
+        self.out_list = numpy_ops[0].out_list
+        for op in numpy_ops[1:]:
+            assert self.in_list == op.in_list, "All ops within OneOf must share the same input configuration"
+            assert self.out_list == op.out_list, "All ops within OneOf must share the same output configuration"
+            assert mode == op.mode, "All ops within a OneOf must share the same mode"
+
+            for inp in op.inputs:
+                inputs.add(inp)
+
+            for out in op.outputs:
+                outputs.add(out)
+
+        NumpyOp.__init__(self, inputs=inputs.union(outputs), outputs=outputs, mode=mode)
+        self.ops = numpy_ops
+
+    def forward(self, data: List[np.ndarray], state: Dict[str, Any]) -> List[np.ndarray]:
+        op_idx = random.randint(0, len(self.ops)-1)
+        data = {key: elem for key, elem in zip(self.inputs, data)}
+        forward_numpyop([self.ops[op_idx]], data, state)
```

Comment:
why not just random.choice(self.ops) rather than going through the idx generation?

Reply:
True. I'll change this.